src/dotnet-interactive-vscode/src/interactiveNotebook.ts

```diff
@@ -136,6 +137,15 @@ export function serializeNotebook(notebook: NotebookFile): string {
     return lines.join('\r\n');
 }
 
+export function languageToCellKind(language: string): CellKind {
+    switch (language) {
+        case 'dotnet-interactive.markdown':
```

Comment:
Do we need our own Markdown language identifier?

Reply:
Yes.  In a polyglot file/cell, that's the classification that gets applied when `#!markdown` or `#!md` is encountered and we need it to be able to properly switch to another language or mark magic commands.